x-pack/plugin/esql/qa/testFixtures/src/main/resources/semantic_text.csv-spec

```diff
@@ -173,3 +173,946 @@ host:keyword | semantic_text_field:semantic_text
+| SORT _id
+;
+
+_id: keyword | result:bool
+1            | true
+2            | false
+3            | null
+;
+
+inequalityWithConstant
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = st_ip != "1.1.1.1"
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:bool
+1            | false
+2            | true
+3            | null
+;
+
+inequalityBetweenFields
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = st_long != st_unsigned_long
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:bool
+1            | false
+2            | true
+3            | null
+;
+
+lessThanWithConstant
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = semantic_text_field < "bye!"
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:bool
+1            | false
+2            | true
+3            | true
+;
+
+lessThanBetweenFields
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = semantic_text_field < st_version
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:bool
+1            | false
+2            | false
+3            | null
+;
+
+
+lessThanOrEqualToWithConstant
+required_capability: semantic_text_type
+
+
+FROM semantic_text METADATA _id
+| EVAL result = semantic_text_field <= "be excellent to each other"
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:bool
+1            | false
+2            | true
+3            | true
+;
+
+lessThanOrEqualToBetweenFields
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = st_integer <= st_long
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:bool
+1            | false
+2            | true
+3            | null
+;
+
+greaterThanWithConstant
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = semantic_text_field > "bye!"
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:bool
+1            | true
+2            | false
+3            | false
+;
+
+greaterThanBetweenFields
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = semantic_text_field > st_version
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:bool
+1            | true
+2            | true
+3            | null
+;
+
+greaterThanOrEqualToWithConstant
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = semantic_text_field >= "be excellent to each other"
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:bool
+1            | true
+2            | false
+3            | true
+;
+
+greaterThanOrEqualToBetweenFields
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = st_integer >= st_long
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:bool
+1            | true
+2            | false
+3            | null
+;
+
+isNull
```

Comment:
tests for [ES|QL predicates](https://www.elastic.co/guide/en/elasticsearch/reference/master/esql-functions-operators.html#esql-predicates)